public/styles/hljs/overrides.scss

```
@@ -1,46 +1,62 @@
-pre {
-  overflow: visible !important;
-  position: relative !important;
+// could remove placeholder selector if we take out
+// PRIMER-REMOVE-ME
+%markdown-pre {
+  overflow: visible;
+  position: relative;
   .btn-group {
-    border: 1px #e5e5e5 solid !important;
-    border-radius: 3px !important;
-    font-size: 14px !important;
-    font-weight: 500 !important;
-    position: absolute !important;
-    right: 5px !important;
-    top: 5px !important;
+    border: 1px #e5e5e5 solid;
+    border-radius: 3px;
+    font-size: 14px;
+    font-weight: 500;
+    position: absolute;
+    right: 5px;
+    top: 5px;
 
     button {
-      border-right: 1px #e5e5e5 solid !important;
+      border-right: 1px #e5e5e5 solid;
     }
+
     button:last-child {
-      border-right: none !important;
+      border-right: none;
     }
   }
 
   .btn-launch, .btn-clipboard {
-    background-color: #f8f8f8 !important;
-    border: none !important;
-    outline: 0 !important;
-    padding: .25rem .5rem !important;
-    transition: 0.2s !important;
+    background-color: #f8f8f8;
+    border: none;
+    outline: 0;
+    padding: .25rem .5rem;
+    transition: 0.2s;
 
     &:hover {
-      background-color: #eee !important;
+      background-color: #eee;
     }
 
     &:active {
-      background-color: #dfdfdf !important;
+      background-color: #dfdfdf;
     }
   }
 }
+
+.markdown-body pre {
+  @extend %markdown-pre;
+  .PRIMER-REMOVE-ME & {
+    @extend %markdown-pre;
+  }
+}
+
 .hljs {
+  padding: $spacer5 $spacer3;
```

The current paddings look for me very big and take the big space between content. I propose to use the new Primer npm padding what's small and beautiful. (The new padding is 24px 12px). Also, it helps with migrating 🙈

```suggestion
  padding: $spacer-5 $spacer-3;
```

---

Seems like these values are better since they're still big enough to avoid overlap with the top buttons. Thanks!